frontend/public/components/cluster-settings/global-config.tsx

```diff
@@ -134,23 +134,21 @@ class GlobalConfigPage_ extends React.Component<GlobalConfigPageProps, GlobalCon
     });
     return (
       <>
-        {!loading && (
-          <div className="co-m-pane__filter-bar co-m-pane__filter-bar--with-help-text">
-            <div className="co-m-pane__filter-bar-group co-m-pane__filter-bar-group--help-text">
+        <div className="co-m-pane__body">
+          {!loading && (
+            <div>
```

Comment:
```suggestion
            <>
```
as you don't need an HTML element here.